dataloader.py

```diff
@@ -109,8 +116,8 @@ def _resize_data(self, image, mask):
         """
         Resizes images to specified size.
         """
-        image = tf.image.resize(image, [self.image_size, self.image_size])
-        mask = tf.image.resize(mask, [self.image_size, self.image_size], method='nearest')
+        image = tf.image.resize(image, self.image_size)
+        mask = tf.image.resize(mask, self.image_size)
```

Comment:
For resizing masks, we should always use the nearest neighbour method, as masks should have integer values, and bilinear interpolation will result in float values.